nevergrad/functions/pownet/solver.py

```diff
@@ -0,0 +1,251 @@
+import os
```

Comment:
if this is all based on Pyomo, you should inclide it in the pyomo directory, that will help structure the code.